lib/logstash/plugin_mixins/elasticsearch/common.rb

```diff
@@ -15,51 +18,6 @@ module Common
     # will never succeed.
     VERSION_TYPES_PERMITTING_CONFLICT = ["external", "external_gt", "external_gte"]
```

Comment:
I don't think this needs to be in common anymore: it will be used in existing ES output, but not in data streams.

It feels like the [event_action_tuple](https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/efc1c069d04444f77db1549d95e4f06131497f11/lib/logstash/plugin_mixins/elasticsearch/common.rb#L33) method doesn't really belong in common as most of it is around doing stuff that data streams wont do: parent/child, update, version,etc.

Reply:
ok let me double-check for `event_action_tuple `

Reply:
I moved the whole `event_action_tuple` + friends out of commons.

Reply:
searching for `VERSION_TYPES_PERMITTING_CONFLICT` in all plugins or logstash repo I can't fin any reference/usage of that. Will remove.